docs/en/operations/_backup.md

```diff
@@ -171,13 +180,118 @@ end_time:          2022-08-30 09:21:46
+
+```sql
+BACKUP TABLE data TO S3('https://mars-doc-test.s3.amazonaws.com/backup-S3/my_backup', 'ABC123', 'Abc+123')
+```
+
+```response
+┌─id───────────────────────────────────┬─status─────────┐
+│ de442b75-a66c-4a3c-a193-f76f278c70f3 │ BACKUP_CREATED │
+└──────────────────────────────────────┴────────────────┘
+```
+
+### Add more data
 
-It is possible to `BACKUP`/`RESTORE` to S3, but this disk should be configured
-in a proper way, since by default you will need to backup metadata from local
-disk to make backup full.
+Incremental backups are populated with the difference between the base backup and the current content of the table being backed up.  Add more data before taking the incremental backup:
+
+```sql
+INSERT INTO data SELECT *
+FROM generateRandom('key Int, value String, array Array(String)')
+LIMIT 100
+```
+### Take an incremental backup
+
+This backup command is similar to the base backup, but adds `SETTINGS base_backup` and the location of the base backup.  Note that the destination for the incremental backup is not the same directory as the base, it is the same endpoint with a different target directory within the bucket.  The base backup is in `my_backup`, and the incremental will be written to `my_incremental`:
+```sql
+BACKUP TABLE data TO S3('https://mars-doc-test.s3.amazonaws.com/backup-S3/my_incremental', 'ABC123', 'Abc+123') SETTINGS base_backup = S3('https://mars-doc-test.s3.amazonaws.com/backup-S3/my_backup', 'ABC123', 'Abc+123')
+```
+
+```response
+┌─id───────────────────────────────────┬─status─────────┐
+│ f6cd3900-850f-41c9-94f1-0c4df33ea528 │ BACKUP_CREATED │
+└──────────────────────────────────────┴────────────────┘
+```
+### Restore from the incremental backup
+
+This command restores the incremental backup into a new table, `data3`.  Note that when an incremental backup is restored, the base backup is also included.  Specify only the incremental backup when restoring:
+```sql
+RESTORE TABLE data AS data3 FROM S3('https://mars-doc-test.s3.amazonaws.com/backup-S3/my_incremental', 'ABC123', 'Abc+123')
+```
+
+```response
+┌─id───────────────────────────────────┬─status───┐
+│ ff0c8c39-7dff-4324-a241-000796de11ca │ RESTORED │
+└──────────────────────────────────────┴──────────┘
+```
+
+### Verify the count
+
+There were two inserts into the original table `data`, one with 1,000 rows and one with 100 rows, for a total of 1,100. Verify that the restored table has 1,100 rows:
+```sql
+SELECT count()
+FROM data3
+```
+```response
+┌─count()─┐
+│    1100 │
+└─────────┘
+```
+
+### Verify the content
+This compares the content of the original table, `data` with the restored table `data3`:
+```sql
+SELECT throwIf((
+        SELECT groupArray(tuple(*))
+        FROM data
+    ) != (
+        SELECT groupArray(tuple(*))
+        FROM data3
+    ), 'Data does not match after BACKUP/RESTORE')
```

Comment:
This looked good in the test but for me it seems a bit excessive for the documentation.
I mean you don't have to verify the count or the contents to use backups if you don't want to.

Reply:
I agree, not necessary, but coming from ops it is something that I always did when setting up a new process, and it did come up on the webinar just now (actually the question was about verifying the checksums, do you know if we can do that- I would rather have the checksums tested instead of selecting all of the data?)